CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
 ## Unreleased
 ### Changes
-- ([\#74](https://github.com/forbole/juno/pull/74)) Added database block count to prometheus to improve alert monitoring
-- ([\#75](https://github.com/forbole/juno/pull/75)) Allow modules to handle MsgExec inner messages
-- ([\#76](https://github.com/forbole/juno/pull/76)) Return 0 as height for `GetLastBlockHeight()` method while no block is saved
 - Updated cosmos/cosmos-sdk to `v0.45.8`
-- Use `sqlx` instead of `sql` while dealing with a PostgreSQL database
+- ([\#74](https://github.com/forbole/juno/pull/74)) Added database block count to prometheus to improve alert monitoring
 - ([\#75](https://github.com/forbole/juno/pull/75)) Allow modules to handle MsgExec inner messages
+- ([\#79](https://github.com/forbole/juno/pull/79)) Use `sqlx` instead of `sql` while dealing with a PostgreSQL database
```

Comment:
```suggestion
- ([\#76](https://github.com/forbole/juno/pull/76)) Return 0 as height for `GetLastBlockHeight()` method if there are no blocks saved in database
- ([\#79](https://github.com/forbole/juno/pull/79)) Use `sqlx` instead of `sql` while dealing with a PostgreSQL database
```